hdfs-scheduler/src/test/java/org/apache/mesos/hdfs/TestStateMachine.java

```diff
@@ -0,0 +1,154 @@
+package org.apache.mesos.hdfs;
```

Comment:
all these tests live in the wrong package space.  they should have parity to the main target class.  So TestStateMachine would be under org.apache.mesos.hdfs.state